x86_64-bootloader.json

```
@@ -5,8 +5,9 @@
     "linker": "rust-lld",
     "pre-link-args": {
         "ld.lld": [
-	    "--script=linker.ld"
-	]
+	        "--script=linker.ld",
+            "--gc-sections"
+	    ]
```

```suggestion
            "--script=linker.ld",
            "--gc-sections"
        ]
```
replacing tabs with spaces

---

Thanks!